implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Could you explain why class has "package" access modifier? 
Could you explain why methods in that class has "static" keyword?